dotnet/src/Connectors/Connectors.HuggingFace/HuggingFaceKernelBuilderExtensions.cs

```diff
@@ -114,4 +116,54 @@ public static class HuggingFaceKernelBuilderExtensions
         return services.AddKeyedSingleton<ITextEmbeddingGenerationService>(serviceId, (serviceProvider, _) =>
             new HuggingFaceTextEmbeddingGenerationService(model, endpoint, apiKey, HttpClientProvider.GetHttpClient(serviceProvider)));
     }
+
+    /// <summary>
+    /// Adds an Hugging Face text embedding generation service with the specified configuration.
+    /// </summary>
+    /// <param name="builder">The <see cref="IKernelBuilder"/> instance to augment.</param>
+    /// <param name="model">The name of the Hugging Face model.</param>
+    /// <param name="apiKey">The API key required for accessing the Hugging Face service.</param>
+    /// <param name="endpoint">The endpoint for the text embedding generation service.</param>
+    /// <param name="serviceId">A local identifier for the given AI service.</param>
+    /// <param name="httpClient">The HttpClient to use with this service.</param>
+    /// <returns>The same instance as <paramref name="builder"/>.</returns>
+    public static IKernelBuilder AddHuggingFaceImageToText(
+        this IKernelBuilder builder,
+        string model,
+        string? apiKey = null,
+        Uri? endpoint = null,
```

Comment:
In this extension method, `apiKey` parameter is followed by `endpoint`, but in `HuggingFaceImageToTextService` constructor, `endpoint` goes first, and then `apiKey`. I would suggest keeping parameters in constructors and related extension methods in the same order for consistency. Even though they have different types (string and Uri) and it's harder to pass them incorrectly, it's still may be confusing.

Reply:
I agree and the current code already had a different pattern will adjust it in the extensions methods, this will introduce a breaking change but will make everything consistent moving forward.